types/electron-progressbar/electron-progressbar-tests.ts

```diff
@@ -6,11 +6,20 @@ const progressBar = new ProgressBar({
         webPreferences: {
             nodeIntegration: false
         }
-    }
+    },
+    indeterminate: false
 });
 
-progressBar.value = 10;
-
 progressBar.on('ready', () => {
     return;
 });
+
+progressBar.on('progress', () => {
+    progressBar.detail = 'progress called 1st time without values';
+});
+
+progressBar.on('progress', (value: any) => {
```

Comment:
Why an `any` type here. The test should reflect that the parameter is a number.

Reply:
Sorry, you're right. (After the n-th typo, I pasetd the code from the editor just to make the n-th +1)